lib/messages.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Messages do
+  def get_prompt(prompt) do
+    messages = %{
```

Comment:
It's fine to have a static mapping like this, but I'm not sure keeping it within this function is the right way to go. Maybe lookup how elixir suggests handling static code. I'm not 100% sure but perhaps something like a module variable or something could be better? I don't know and you could definitely change ,my mind. 😸